source/hardware/controllers/reloop_digital_jockey_2_interface_edition.rst

```diff
@@ -0,0 +1,58 @@
+.. _reloop_digital_jockey_2_interface_edition:
+
+Reloop Digital Jockey 2 Interface Edition
+=========================================
+
+The *Reloop Digital Jockey 2 Interface Edition* is a 2 deck USB MIDI controller designed to work with Traktor software. Compared to *Controller Edition* the *Interface Edition* is additionally equipped with a high quality 4-channel USB audio interface and touch-sensitive jog wheels. The microphone input is available to the computer for recording and broadcasting. The controller can be powered only from the USB bus, however the included external power supply can serve as the power source too. Using a separate power supply has no impact on the output level or LED brightness.
```

Comment:
Is it the Line/Phono input or the Mic input which can be routed to the software?
The SW / MASTER switch is next to the Line...
I think this is worth mentioning in the Hardware paragraph.

Reply:
There is no SW/MASTER switch in "Interface Edition". You are talking about "Master Edition" which is a third variant of this controller.
On this controller there is no control over routing.

Reply:
Hmm,.okay then. I saw [this image](https://www.reloop.com/media/catalog/product/cache/9/image/768x768/9df78eab33525d08d6e5fb8d27136e95/2/2/4/224092_Reloop_BA.png) on the official Reloop site, guess they mixed it up with the ME then

Reply:
Yeah, this is strange. At this position I have MIDI SWITCH (M1/M2).